core/vtk/ttkPlanarGraphLayout/ttkMergeTreeVisu.h

```diff
@@ -0,0 +1,1089 @@
+/// \ingroup vtk
+/// \class ttkMergeTreeVisu
```

Comment:
rename to ttkMergeTreeVisualization (or ttkMergeTreeViz, but I believe being more explicit is better).

Reply:
Done!